xAPI.md

```diff
@@ -298,11 +298,46 @@ are based on the specification set described below. For this reason, sections th
 _high-level overview_ of a given facet of the Experience API are labeled **description** or 
 **rationale**. Items in this document labeled as **requirements**, **details** or **examples** are more technical.
 
+<a name="def-must-should-may" />
+### 2.1 MUST / SHOULD / MAY 
+Three levels of obligation with regards to conformance to the xAPI 
+specification identified by the terms MUST, SHOULD and MAY. A system that fails to implement a MUST 
+(or a MUST NOT) requirement is non-conformant. Failing to meet a SHOULD requirement is not a violation of conformity, 
+but goes against best practices. MAY indicates an option, to be decided by the developer with no consequences for conformity. 
```

Comment:
instead of "best practices" maybe just say "the recommendations of the specification"?